### ISSUE TITLE
new logo

### ISSUE THREAD

**Issue by notpeter** [notpeter/playdate-luacats#2]
<img width="535" alt="luacat" src="https://github.com/notpeter/playdate-luacats/assets/145113/f3bc4087-844b-4856-89df-4d2780ec69b1">

**Comment by psychicteeth** [comment]
Genuinely better than the horrible AI slop one you have on the front page at the moment.

**Comment by notpeter** [comment]
That's the joke.

**Comment by psychicteeth** [comment]
I apologise for my earlier comment which was very rude. To make up for it I have made a logo, please feel free to use it.
https://github.com/notpeter/playdate-luacats/pull/9

**Comment by psychicteeth** [comment]
Inverted version that preserves Lua's logo shading

small
<img width="143" height="140" alt="Image" src="https://github.com/user-attachments/assets/8cf8f080-0ac3-43ae-b75d-f0b4628103cf" />

large
<img width="858" height="840" alt="Image" src="https://github.com/user-attachments/assets/80e6e93b-085b-4a96-8dac-8d9e68f75e4a" />

Happy to do more modifications if they're desired!
Thanks and again sorry

**Comment by burnttoastdev** [comment]
This logo is so adorable!

**Comment by notpeter** [comment]
That's a nifty logo.
I definitely prefer the version in this issue (left) to the one in #9 (right)
I don't love how it looks on github though since the transparency gets rendered as off-white.

<img width="927" height="528" alt="Image" src="https://github.com/user-attachments/assets/74270c23-b5cb-4a83-b6e3-c9a261a444f9" />

Also would love to have something that looks good in the link card. (shitty mock)

<img width="1280" height="640" alt="Image" src="https://github.com/user-attachments/assets/03df5dce-2924-426c-9743-ea0c5676128d" />

I'm extra motivated since I managed to erase the old link card while playing around with this. Oops.

**Comment by burnttoastdev** [comment]
Yes I do agree the one on the left looks a lot better, also the mockup you made looks pretty good.
